### PR TITLE
[READY] Metastation: The Floofening - Let's make our Metastation be the prettiest out there!

### PR DESCRIPTION
**Pull request by KathrinBailey**
## TL;DR
make meta look doubleplusgood

## About The Pull Request
There's so many changes that I'm probably going to miss one or two. Check changelog to see them all.
Metastation has been given a facelift. I started this with minor edits to dorms, a complete deletion of security and the prison with a rebuild of that entire department from scratch. Now science is getting a facelift and medical a partial facelift. Bar and kitchen have been rebuilt.

**This has been tested and is ready for a merge.**

## Why It's Good For The Game
Judge for yourself in the media section.
This is a visual improvement to many parts of Metastation. I don't want TG's Metastation to have a better service area than ours. The bar has been expanded massively for all those people who like bar-building. The overall character of science and medical remains completely unchanged, however, multiple rooms are now brought to higher standards visually and have been reworked, where possible, to be more roleplay-friendly.

**Why service is better:**
The old service area was probably tied with Deltastation for the worst bar and bar seating area there was. It was awful. It looked awful. It barely served its function. The bar has been expanded into the threatre, there is more seating and an extra bar entrance. With all the extra space to the right of the bar, players can build their own theatre if they want it. I'm not 100% happy with the bar, but all feedback I've been given has been positive.

**Why medical is better:**
The cloning room is much larger, meaning that when there's a pile-up of bodies, everything can still be kept neatly organised. Electrochromatic windows allow medical staff to give freshly-cloned people some privacy. They'll emerge naked and, if doctors choose to, they won't be oogled by the entire department. Clothing vendors and a bed behind curtains help with cloned characters retaining at least _some_ dignity. Genetics now can work and man their desk at the same time. Their workspace looks nicer and they have a more humane (and nicer looking) pen for monkeys.

**Why science is better:**
Roundstart nanites is something TG has done and is definitely worth a trial. Robotics is less needlessly huge and after consulting robotics mains and science mains, they enjoyed the much neater and cleaner look of robotics as well as the surgery room. Also, robotics surgery has an air scrubber that is set to filter miasma. The RD office has been rebuilt and looks much much nicer than before. It has nicer flooring, it's more neatly organised, their desk is nicer, their computer setup is neat. They get flowers (yay!) and their display case is proudly on display in the centre of the room.
Also, toxins now has the appropriate gases.
The pipe dispenser in toxins is something that people have wanted since forever. They might have wanted a RPD instead, but a pipe dispenser is cool. Instead of roundstart unwrenching the existing piping, players can construct their own.
It just generally looks much nicer.

## Changelog
:cl: Kathrin Morrison / Floof Ball#0798
add: Pipe dispenser to toxins. You now don't need to rush Aux Base roundstart, pester Engineering or hack the science autolathe. Pipe dispensers are almost never used, so unused content is used.
add: Nanite programming room for science. It's small and needs upgrading.
add: A bed and a windows that can be toggled to block sight for privacy. Your naked body shouldn't be oogled by the entire department and everyone in it.
add: Goddamned clothing vendors into cloning because LRP man bad. Some excellent roleplaying opportunities for people who use cloning.
add: New monkey pen for genetics that makes more sense. Cuffing animals to beds will scare the shit out of them and add extra variables into test results that will affect the outcome of some tests.
add: New Aft and Starboard Quarter Maintenance rooms as science has been moved around a lot, featuring: Ghetto Botany that actually makes sense! No dirt piles on plating. Circuit board room, it's just full of mostly useless and completely harmless circuit boards. This room serves as the DIY room I added on Box, just without the APC. A distro refilling station.
add: A diner. It's space that needed to be filled. If there's any new civilian jobs in the future, their office can go here.
tweak: Revamped circuity lab.
tweak: Revamped toxins and gas storage.
tweak: Revamped psychologist's office.
tweak: Revamped maint bar.
tweak: Revamped Captain's Quarters. Someone complained that I made the prison look too nice compared to the rest of the station and that it was nicer than the Captain's Quarters.
tweak: Moved science servers.
tweak: Moved Xenobiology up, it's less of a huge obstacle in space now.
tweak: Moved Starboard Quarter Solars, Toxins launch and Circuity further to the right.
tweak: Clone office is now in arrivals.
tweak: Rebuilt Research Director's office.
tweak: Rebuilt robotics. The surgery room has been significantly improved. It's a more sombre and private experience.
tweak: Rebuilt science break room. It now has a small changing room and a seating area so people can actually take breaks.
tweak: Rebuilt cloning. Looks significantly better. It's roomier and has privacy options. The extra space given will help significantly with organising when there's a huge pile-up of corpses to be cloned. Extra space is given for extra cloning pods.
tweak: Rebuilt genetics; genetics desk has been removed as it's absolutely pointless and is never used. Instead, genetics is wider and encompasses the desk.
tweak: Rebuilt arrivals bathroom to be a species segregated bathroom. It's nicer.
tweak: Slight modification to prison loot for minor ghetto chem potential.
tweak: Instruments added to maint. This appears on Box and aside from maybe cargo (good luck), was the only way you could get instruments.
tweak: Camera network name changes in certain areas to be more suitable.
balance: Optimised the amount of gas cans in toxins storage to be actually useful. Less used gases are fewer in supply and more used gases are more common. Also, six TTVs has gone down to five.
fix: Inconsistent decals in security.
fix: Station ruins areas are now in appropriate locations.
fix: Blast doors in some windows where they don't belong. Toxins keeps their blast doors because, well, it's toxins.
fix: Mail sorting now works for security.
fix: Window mispalced in the interrogation room looking at a shower curtain.
del: Science firing range. It never gets used and it was replaced with nanites.
del: Theatre. Goodbye dead unused content! :)
/:cl:

## Media
image
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by Azarak** [comment]
I'd like if people could voice their opinons about this.

I dont see connector ports to freezers or heaters in toxins mixing, nor I don't really know how piping looks underneath, how much space you've got to pipe your own stuff.

I think the bar table could be moved 1 tile to the back, so instead people have 2 tiles to freely move around the bar, instead of the bartender having 2 tiles.

I dont know about trying to stuff sofas or plants into a lot of the areas, I understand araes that are important offices or places of visitation (det's office, psychologist office). But putting them in toxins or nanites labs feels kind of like they're just there just to be. Robotics is understandable as I guess people hang out there often.

I don't like the robotics scrubber being automatically set to scrub miasma, it let's people have bad habits such as letting corpses rot out, instead of handing them over to medical, and also not having to deal with the fallout of miasma, compared to having to call in an engineer/atmo tech/AI.

**Comment by KathrinBailey** [comment]
> I'd like if people could voice their opinons about this.
> 
> I dont see connector ports to freezers or heaters in toxins mixing, nor I don't really know how piping looks underneath, how much space you've got to pipe your own stuff.

image

Idk if this causes problems with there being no pipe connected, the point is you put your own down, you have a ton of space to do so since distro has been moved out of the way.
